include/odp/api/spec/crypto.h

```diff
@@ -122,12 +122,26 @@ typedef enum {
 	 */
 	ODP_AUTH_ALG_SHA256_HMAC,
 
+	 /** HMAC-SHA-384
+	 *
+	 *  SHA-384 algorithm in HMAC mode
+	 */
+	ODP_AUTH_ALG_SHA384_HMAC,
```

Comment:
In addition to this enum, odp_crypto_auth_algos_t bit field needs to be updated with .sha384_hmac.